Sources/EdgeNetworkHandlers/NetworkResponseHandler.swift

```diff
@@ -263,6 +263,13 @@ class NetworkResponseHandler {
     /// If handle is of type "state:store" persist it to Data Store
     /// - Parameter handle: current `EventHandle` to store
     private func handleStoreEventHandle(handle: EdgeEventHandle) {
+        let storeResponsePayloadManager = StoreResponsePayloadManager(EdgeConstants.DataStoreKeys.STORE_NAME)
+        // handling async scenario when in-flight response is handled after the privacy status update was handled by the Edge extension
+        if Edge.currentPrivacyStatus == .optedOut {
+            storeResponsePayloadManager.deleteAllStorePayloads()
```

Comment:
Do we need to delete here or just return?  Seems like the delete is overkill, because the actual delete would've already been processed by handleConfigurationResponse.

Reply:
not necessary indeed